README.md

```diff
@@ -64,6 +64,7 @@ git checkout refresh  # Branch for our staging-stable server.
 
 #### Updating all dependencies
 
+`setup.sh` will install any updated dependecies and rebuild the site's assets.
 Each time you fetch from the upstream repository (this repo),
 you should install and update dependencies by running `./setup.sh`.
```

Comment:
This reads as a bit redundant by taking about updating the dependencies twice in the paragraph. How about something like:
```
Each time you fetch from the upstream repository (this repo), run `./setup.sh`. This setup script will remove and re-install the project dependencies and rebuild the site's JavaScript and CSS assets.
```